### PR TITLE
Stop storyboard visual tests from crashing

### PR DESCRIPTION
**Pull request by nyquillerium**
Also adds some documentation to "Passing" in DrawableStoryboard

### CODE REVIEW

File: osu.Game.Tests/Visual/TestCaseStoryboard.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ namespace osu.Game.Tests.Visual
     [TestFixture]
     public class TestCaseStoryboard : OsuTestCase
     {
-        private readonly Container<DrawableStoryboard> storyboardContainer;
```
Comment (peppy) [inline osu.Game.Tests/Visual/TestCaseStoryboard.cs]: Can you explain what was going on here to cause a crash from this?

**Comment by peppy** [comment]
As discussed IRL, this probably won't fix the underlying issue (cross-thread method call)

**Comment by peppy** [comment]
Can you add an explanation of how this fixes the issue?

**Comment by nyquillerium** [comment]
Actually, this seems to have since stopped happening on current master. Going to close this as this is probably not the solution

**Comment by peppy** [comment]
Did you find where the fix was?

**Comment by nyquillerium** [comment]
Currently still looking, though if there's more adjustments needed it would most likely not come from this PR.

**Comment by nyquillerium** [comment]
This was resolved by https://github.com/nyquillerium/osu/commit/30815ace62cc0580f9868999c6cf5a3597110638

**Pull request closed without merging**